buildSrc/src/main/java/com/uber/okbuck/core/util/FileUtil.java

```diff
@@ -82,10 +85,16 @@ public static boolean isZipFile(File file) {
     }
   }
 
-  public static void symlink(Path link, Path target) {
+  public static void symlink(Path link, @Var Path target) {
     try {
       LOG.info("Creating symlink {} -> {}", link, target);
-      Files.createSymbolicLink(link, target);
+      if (System.getProperty("os.name").startsWith("Windows")) {
```

Comment:
This should be ideally used to create a symlink creator factory and set at the very start inside the okbuck plugin instead of checking every single time. We dont expect the os to change mid invocation